exchanges/orderbook/depth_test.go

```diff
@@ -910,3 +896,37 @@ var movementTests = []struct {
 			{[]any{20.0, true}, Movement{NominalPercentage: 0.7105459985041137, ImpactPercentage: FullLiquidityExhaustedPercentage, SlippageCost: 190.0, FullBookSideConsumed: true}},
 		}},
 }
+
+func TestGetPair(t *testing.T) {
+	t.Parallel()
+	depth := NewDepth(id)
+	require.Empty(t, depth.GetPair())
+	depth.pair = currency.NewPair(currency.BTC, currency.WABI)
+	require.Equal(t, depth.pair, depth.GetPair())
+}
+
+func TestGetAsset(t *testing.T) {
+	t.Parallel()
+	depth := NewDepth(id)
+	require.Empty(t, depth.GetAsset())
+	depth.asset = asset.Spot
+	require.Equal(t, depth.asset, depth.GetAsset())
+}
+
+func TestGetExchange(t *testing.T) {
+	t.Parallel()
+	depth := NewDepth(id)
+	require.Empty(t, depth.GetExchange())
+	depth.exchange = "test"
+	require.Equal(t, depth.exchange, depth.GetExchange())
+}
+
+func TestGetKey(t *testing.T) {
+	t.Parallel()
+	depth := NewDepth(id)
+	require.Empty(t, depth.GetKey())
+	depth.exchange = "test"
+	depth.pair = currency.NewPair(currency.BTC, currency.WABI)
+	depth.asset = asset.Spot
+	require.Equal(t, key.ExchangePairAsset{Exchange: depth.exchange, Base: depth.pair.Base.Item, Quote: depth.pair.Quote.Item, Asset: depth.asset}, depth.GetKey())
```

Comment:
```suggestion
require.Equal(t, 
		key.ExchangePairAsset{Exchange: depth.exchange, Base: depth.pair.Base.Item, Quote: depth.pair.Quote.Item, Asset: depth.asset}, 
		depth.GetKey())
```
A little more readable

Reply:
🚀